src/explorer.ts

```diff
@@ -183,11 +198,13 @@ class KubernetesResourceFolder extends KubernetesFolder {
             host.showErrorMessage(childrenLines.error[0]);
             return [new DummyObject("Error")];
         }
+        if (this.kind.abbreviation === "pod") {
+            const parsedPods = parsePodLine(childrenLines.result);
+            return parsedPods.map((line) => {
+                return new KubernetesResource(this.kind, line.name, { status: line.status });
```

Comment:
Interesting, I would have thought the header names were capitalised - does that not matter?

Reply:
I changed the header check to be case sensitive, will change this as the check for status has to be case insensitive.